grails-app/controllers/org/pih/warehouse/SecurityInterceptor.groovy

```diff
@@ -128,5 +128,7 @@ class SecurityInterceptor {
             redirect(controller: 'dashboard', action: 'chooseLocation')
             return false
         }
+
+        return true
```

Comment:
@mdpearson this is the issue that caused the blank page (not anything related to the bundle or context path as we expected). I think I accidentally removed it while rebasing. I fixed a few other issues I run into. 

cc @jmiranda

Reply:
nice find! 🔍